internal/providers/terraform/aws/cloudwatch_log_group.go

```diff
@@ -1,84 +1,18 @@
 package aws
 
 import (
+	"github.com/infracost/infracost/internal/resources/aws"
 	"github.com/infracost/infracost/internal/schema"
-	"github.com/shopspring/decimal"
 )
 
 func GetCloudwatchLogGroupItem() *schema.RegistryItem {
```

Comment:
Might not be possible, but can we update these to be internal functions (`getCloudwatchLogGroupItem`). We've started using that for new resources.

Reply:
I think it's easy, I'll applyt it.